vizro-core/changelog.d/20231218_204302_antony.milne_improve_dashboard_title.md

```diff
@@ -0,0 +1,47 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Added
+
+- When set, the dashboard title appears alongside the individual page title as the text labelling a browser tab. ([#228](https://github.com/mckinsey/vizro/pull/228))
+
```

Comment:
This is meant to be a "non-expert user understandable" way of saying `<title>` and is inspired by the description of Dash's `title` argument.

Reply:
Just wondering whether we should align the naming - I think we refer to it as dashboard title while Dash refers to it as app title. Dashboard title makes more sense to me as it's an argument in the dashboard model, but under the hood we seem to overwrite the app title

This might also be just worth mentioning in the docs for people coming from Dash

Reply:
Yeah, I think "dashboard title" is best here because currently the only way a user might set their title is through `Dashboard.title` since we don't advertise the `Vizro(title=...)` method anywhere.

So I will leave this alone but add something to the docs that clarifies.